warp-academy-docs/standards/atomic-asset.md

```diff
@@ -0,0 +1,121 @@
+# Introduction
+
+## What is atomic asset?
+
+An atomic asset is the concept, where **asset** (like an image) and **smart contract** are deployed atomically - in one logical transaction.
+It is possible, thanks to unique properties of arweave blockchain. In a typical blockchain asset is saved on some storage (for example IPFS) and only smart contract is deployed on-chain (for example ERC721). Why to choose atomic asset:
```

Comment:
Typically, a blockchain asset is stored on a storage system (for example, IPFS), and only a smart contract is deployed on-chain (for example, the ERC721 standard).